Add "ai.onnx.pytorch" onnx domain

bddppq opened the pull request:
@zrphercule

bddppq committed:
Add pytorch onnx domain

houseroad reviewed (approved):
Can we change `ai.onnx.pytorch` to something `torch_op`? Looks a little bit weird to have `.` in the folder name. Also android building fails. Please fix that.

Since we start another domain, I think we should start from 1 :-)

Prestamped.

bddppq commented:
1. renamed the directory to "torch_ops"
2. android build failure was because in mobile build we don't link the whole onnx lib (just onnx_proto), so schema is not available.
3. changed the version to 1

bddppq committed:
Test whether ONNX_API macro can fix windows error

bddppq committed:
update submodule

bddppq committed:
update submodule

bddppq committed:
update submodule

bddppq committed:
set ONNX_BUILD_MAIN_LIB

bddppq committed:
.

bddppq committed:
revert submodule change

bddppq committed:
remove c++ test

bddppq committed:
clang-format

bddppq committed:
don't count into dbg static schema number
